src/cnv/pyproject.toml

```diff
@@ -17,16 +17,19 @@ readme = "README.cnv.md"
 [project.license]
 text = "Apache-2.0"
 
-[project.scripts]
-run_tests = "pytest:main"
+[tool.uv.sources.ugbio_core]
+workspace = true
```

Comment:
What does this workspace flag mean?
What does it mean when it set to false?

Reply:
When set to "true" it means that the dependency should be provided by the workspace, rather than fetched from PyPI or another registry (see here: https://docs.astral.sh/uv/concepts/workspaces/#workspace-sources )